graphsync_test.go

```diff
@@ -383,3 +384,73 @@ func TestGraphsyncRoundTrip(t *testing.T) {
 		}
 	}
 }
+
+func TestRoundTripLargeBlocksSlowNetwork(t *testing.T) {
```

Comment:
I think this test case deserves a comment explaining the kind of problem it's defending against